applications/CompressiblePotentialFlowApplication/compressible_potential_flow_application_variables.cpp

```diff
@@ -11,12 +11,20 @@ KRATOS_CREATE_VARIABLE(double, ADJOINT_VELOCITY_POTENTIAL)
 KRATOS_CREATE_VARIABLE(double, ADJOINT_AUXILIARY_VELOCITY_POTENTIAL)
 
 // Flow field magnitudes
-KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS(VELOCITY_INFINITY)
 KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS(VELOCITY_LOWER)
 KRATOS_CREATE_VARIABLE(double, PRESSURE_LOWER)
 KRATOS_CREATE_VARIABLE(double, POTENTIAL_JUMP)
 KRATOS_CREATE_VARIABLE(double, ENERGY_NORM_REFERENCE)
 KRATOS_CREATE_VARIABLE(double, POTENTIAL_ENERGY_REFERENCE)
+KRATOS_CREATE_VARIABLE(double, CP)
```

Comment:
If this is the pressure coefficient, there is a variable in `cfd_variables.h` for it.

Reply:
Done! One question: isnt it kind of annoying that we need to include this header in all of the files where we want to use these variables? Specially when cfd_variables.h includes another bunch of headers, some of which are not even used:

```c++
// System includes
#include <string>
#include <iostream>

// External includes


// Project includes
#include "includes/define.h"
#include "containers/variable.h"
#include "containers/variable_component.h"
#include "containers/vector_component_adaptor.h"
#include "includes/kratos_components.h"
#include "includes/ublas_interface.h"
#include "containers/array_1d.h"
#include "containers/weak_pointer_vector.h"
#include "containers/periodic_variables_container.h"
```